src/webapp/topology.py

```diff
@@ -185,13 +185,14 @@ def _expand_contactlists(self, contactlists: Dict, authorized: bool) -> Dict:
             contact_data = expand_attr_list(contact_data, "ContactRank", ["Name", "ID", "ContactRank"], ignore_missing=True)
             for contact in contact_data:
                 contact_id = contact.pop("ID", None)  # ID is for internal use - don't put it in the results
-                if authorized and self.common_data.contacts:
-                    if contact_id in self.common_data.contacts.users_by_id:
-                        extra_data = self.common_data.contacts.users_by_id[contact_id]
-                        contact["Email"] = extra_data.email
-                        contact["Phone"] = extra_data.phone
-                        contact["SMSAddress"] = extra_data.sms_address
-                        dns = extra_data.dns
+                if self.common_data.contacts and contact_id in self.common_data.contacts.users_by_id:
+                    user = self.common_data.contacts.users_by_id[contact_id]  # type: User
+                    contact["CILogonID"] = user.cilogon_id
```

Comment:
I guess this is obvious - but the idea here is that the `CILogonID` field does not not require authorization, right?

Reply:
Correct, it's public.